W-16432084 August ga release pr (#48)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/assigning-permissions.adoc

.. In *Setup*, enter `permission sets` in the *Quick Find* box and click *Permission Sets*.
.. Locate *AEH_Anypoint* in the list.

== Assign API Experience Hub Users Approver Permission

> Assign the API Experience Hub User Approver permission to enable a user to approve or reject pending registration requests and enable or disable member users only. This permission doesn't allow this user to perform any other operations for the portal. 

Reviewer: ```suggestion
Approve or reject pending registration requests and enable or disable member users with the API Experience Hub User Approver permission.
```
I suggest a rewording to describe the action first. I also removed the last sentence because Is it necessary to clarify what this permission does NOT allow? Please apply this suggestion only if you agree and think it is correct.

> When accessing the *Manage your API portal* page, the user with this permission can access the following *User Management* pages:

Reviewer: ```suggestion
This permission grants access to the following *User Management* pages inside *Manage your API portal*:
```
Optional suggestion to make this shorter.

* Portal members
+
Enable or disable a user's access to the portal. 
* Pending registration requests
+
Approve or reject a user's registration request.
* Settings
+
View information about the Salesforce account and portal.

To assign this permission to a user:

. Go to *Access Management*.
. Click *Users* and select a username. 
. Select *Permissions* and click *Add permissions*.

> . In the *Select permissions to add for this user* search box, enter *API Experience Hub Users Approver*, select the checkbox, and then click *Next*.

Reviewer: ```suggestion
. In the *Select permissions to add for this user* search box, enter *API Experience Hub Users Approver* and select the checkbox.
. Click *Next*.
```
Minor suggestion to break the step.
